enhancements/installer/nutanix-ahv-ipi.md

```diff
@@ -0,0 +1,183 @@
+  ? Prism Central Username admin@ocp.nutanix.com
+  ? Prism Central Password [? for help] **********
+  ? Cluster (Prism Element) nx01.ocp.nutanix.com
+  ? Container default-container-70593
+  ? Subnet vm-network
+  ? Virtual IP Address for API 192.168.1.5
+  ? Virtual IP Address for Ingress 192.168.1.6
+  ? Base Domain ocp.nutanix.com 
+  ? Cluster Name ocp01
+  ? Pull Secret [? for help] ****
+  ```
+
+- Implement Nutanix specific platform installer customizations:
+  
+  **Platform**
+  
+  - prismCentral (required string): Name or IP address of the Nutanix Prism Central instance to target.
+  - prismCentralUser (required string): Username for Prism Central access (Cluster Admin).
+  - prismCentralPass (required string): Password for Prism central access.
+  - cluster (required string): Nutanix cluster (Prism Element) where the OpenShift cluster will be created.
+  - container (required string): Nutanix container name for OS disk install.
+  - subnet (required string): Nutanix subnet name (IPAM or DHCP Enabled).
+  
+- Provide documentation (in similar format to other supported IPI platforms) to help users use and customize the IPI installer on Nutanix:
+  - Prerequisite instructions prior to invoking installer
+  - Description of installer options and customizations as they apply to Nutanix
+  - Post installation instructions on further customizations a user may wish to apply
+
+### User Stories
+
+Story 1
+As an OpenShift consumer, I want to quickly, with minimal input and default options, be able to use the OpenShift installer to create and destroy an OpenShift 4 cluster on Nutanix AOS with AHV.
+
+Story 2
+As an OpenShift consumer, I want to be able to use the OpenShift installer with customizations (i.e. enhanced security) to create and destroy an OpenShift 4 cluster on Nutanix AOS with AHV.
+
+Story 3
+As an OpenShift platform, I want OpenShift on Nutanix to be maintained and released like other supported OpenShift platforms and covered by CI tests.
+
+### Implementation Details/Notes/Constraints
+
+- Nutanix AHV supports VM guest customization via [cloud-init Config Drive v2](https://cloudinit.readthedocs.io/en/latest/topics/datasources/configdrive.html).
+
+- It will be assumed the customer utilizes a subnet configured with Nutanix IPAM or will provide a properly configured DHCP server that is available on the configured L2 network.
+
+- Nutanix does not provide networking services like DNS or load balancing. With this in mind, we require the ability to automate DNS and load balancing services internal to the cluster. A solution similar to the baremetal-networking enhancement can be utilized.
+
+### Risks and Mitigations
+
+- The Nutanix Cluster API provider is a work-in-progress.  It is currently being tracked internally at Nutanix and expected to be open sourced.
+
+## Design Details
+
+- Cloud Credential Operator (CCO) will operate in manual mode. Possible migration to passthrough mode in future enhancement.
```

Comment:
You might add 'Possible migration to passthrough mode in future enhancement' to the out-of-scope heading

Reply:
It doesn't seem like it is desirable to even move to passthrough, so will remove entirely.